content/ember-4-0-released.md

```diff
@@ -0,0 +1,231 @@
+beta cycle normally takes about six weeks to mature a branch off `master` to
+release quality.
+
+Finally, today we're also promoting Ember 3.28 to become Ember's latest
+Long-Term Support (LTS) release. Ember's LTS release process provides bugfix
+support for about 36 weeks, and security patches for about 54 weeks. See the [3.28 release
+post](https://blog.emberjs.com/ember-3-28-released) and [LTS
+documentation](https://emberjs.com/releases/lts/) for more details.
+
+## Major Releases in Ember
+
+Why are there no new features in this release?
+This has been the tradition for all major Ember releases since v2.
+We believe that major versions should be rare, and that when they happen,
+they should not introduce new features.
+
+This strategy aims to give Ember
+developers the time and space to upgrade dependencies and
+navigate any breaking changes, without needing to
+learn lots of new things at the same time.
+In many libraries, including Ember, a major release means that there are breaking changes.
+And while it would be exciting for major releases to be splashy and full of
+new features,
+we think it is more important to reduce risks for our developers and
+the users that depend on them.
+
+Our hope is that that library maintainers across the JavaScript ecosystem
+will follow this pattern, which makes it easier for developers to keep up
+with security patches and bugfixes in their project's dependencies.
+
+Our plans for Ember 4.0 were originally announced in July 2021 in
+[The Road to Ember 4.0](https://blog.emberjs.com/the-road-to-ember-4-0/).
+
+## Trying Ember for the first time
+
+If you're interested in trying Ember for the
+first time today, get started by running:
+
+```bash
+npx ember-cli new my-project
+cd my-project
+npx ember serve # Then visit http://localhost:4200
+```
+
+## How to upgrade your project to Ember 4
+
+If your app or addon runs with no deprecations in the latest release of `v3.28`
+and you have `v2` of `ember-auto-import` installed, you should be able to
+upgrade to Ember 4 with no additional changes.
+
+Follow these steps in order:
+
+1. Upgrade your project to the latest patch version of `v3.28`.
+   Many developers can do this by running
+   `npx ember-cli-update --to 3.28`. More details about how to upgrade your Ember
+   app/addon are in the
+   [general upgrade guide](https://cli.emberjs.com/release/basic-use/upgrading/#upgradinganemberappitself).
+2. Resolve all deprecation warnings. Deprecated APIs are removed in Ember 4. You may need to upgrade some
+   of your dependencies if they are using deprecated APIs. Applications that need to upgrade through several versions may want to consider
+   the
+   [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow)
+   addon to isolate individual deprecations.
+3. Make sure your app builds successfully.
+4. Install `ember-auto-import` if your app does not already have it:
```

Comment:
I'm not sure this is sufficient to install `ember-auto-import` v2, as it requires webpack also be installed. The README suggests `npm install --save-dev ember-auto-import webpack` Maybe we should point to the `ember-auto-import` README?